.github/workflows/maven-publish.yml

```diff
@@ -0,0 +1,62 @@
+# This workflow will build a package using Maven and then publish it to GitHub packages when a release is created
+# For more information see: https://github.com/actions/setup-java/blob/main/docs/advanced-usage.md#apache-maven-with-a-settings-path
+
+name: Maven Package
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        token: ${{ secrets.GAMEDOORA_PKGS }}
+    - name: Set up JDK 8
+      uses: actions/setup-java@v2
+      with:
+        java-version: '8'
+        distribution: 'adopt'
+#     - name: Download Maven
+#       run: |
+#         curl -sL https://www-eu.apache.org/dist/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.zip -o maven.zip
+#         apt-get update
+#         apt-get -y install unzip
+#         unzip -d /usr/share maven.zip
+#         rm maven.zip
+#         ln -s /usr/share/apache-maven-3.6.3/bin/mvn /usr/bin/mvn
+#         echo "M2_HOME=/usr/share/apache-maven-3.6.3" | tee -a /etc/environment
```

Comment:
@aprajshekhar Is this going to be required in the future? If not I suppose we can remove it

Reply:
@hybridx we might need it if the version supported by action changes. I would like it to be here as a reference, if required in the future.